pysnooper/tracer.py

```diff
@@ -5,6 +5,10 @@
 import sys
 import re
 import collections
+if sys.version.startswith("3"):
+    from collections import ChainMap
+else:
+    from ConfigParser import _Chainmap as ChainMap
```

Comment:
The pythonic way would be using `try` + `except` instead of probing for the Python version.

```python
try:
    from ConfigParser import _Chainmap as ChainMap
except ImportError:
    from collections import ChainMap
```

Reply:
Correct, I fixed it later :)